main/index.ts

```diff
@@ -369,7 +367,7 @@ ipcMain.on('tray:action', (e, action, ...args) => {
   log.info('Tray sent unrecognized action: ', action)
 })
 
-app.on('activate', () => windows.activate())
+app.on('activate', () => windows.showTray())
```

Comment:
this used to call `windows.activate()` -> `api.showTray()` which had a lot more code than the current `showTray()`. is this an OK substitute?

Reply:
showTray now calls `tray.show` which is where all that code went - I just made the API cleaner with a class based approach for dash and tray